packages/data-point-express/lib/factory.test.js

```diff
@@ -16,7 +16,7 @@ describe('create - all middleware', () => {
   beforeAll(() => {
     const options = {
       entities: {
-        'transform:hello': acc => ({
+        'transform:hello': (value, acc) => ({
```

Comment:
`input`

Reply:
its all over the place, and its not that easy change, it does make sense I agree, but would rather leave it for a second round where we actually check on which reducers makes sense to even pass acc at all, since I couldn't run the codemods on the data-point code itself aside from examples, it was 1 day of search and replace one by one

Reply:
OK, fair enough. Fixing something like this is simple enough to do as a first PR, so I'm cool with leaving it for now.